word-guess-1.rb

```diff
@@ -0,0 +1,142 @@
+#Word Guess Game - Sara Stiltner and Lily Sky
+#ADA c9
+#Week 2
+#Pair Programming exercise
+
+#============ GEMS ========
+
+require "colorize"
+require "colorized_string"
+require "awesome_print"
+
+#============ CLASS ========
+
+class Word
+
+  attr_reader :word, :word_array
+  attr_writer :letter
+  attr_accessor :working_word, :word_length
+
+  def initialize(word)
+    @word = word
+  end
+
+  def word_length
+    @word.length
+  end
+
+  def word_to_array
+    word_array = @word.chars
+    return word_array
+  end
+
+  def working_word
+    working_word = []
+    @word.length.times do |i|
+      working_word << "_ "
+    end
+    return working_word
+  end
+end
+
+#============ STORAGE ========
+
+library = ["LOVE", 'CUPID', "HEART", "ROSES", "GALENTINE", "ROMANCE", "HYGGE", "SECURITY", "SWEETHEART", "INSURANCE", "CHOCOLATE", "ATTENTION", "FLOWERS", "ADORATION", "AFFECTION", "HONEY", "DINNER", "DRAMA", "DELUSION", "SOULMATE", "PANCAKES", "COURT", "CRUSH", "SWEETHEART", "CONSENT"]
+
+heart = [ "   .:::.   .:::.   ".colorize(:red), "  :::::::.:::::::  ".colorize(:red), "  :::::::::::::::  ".colorize(:red), "   '::::::::::::'  ".colorize(:red), "     ':::::::'    ".colorize(:red), "        ':'      ".colorize(:red)]
+
+
+
+#============ PLAY GAME ========
+
```

Comment:
A lot of this main game functionality should be broken into methods and potentially a game class.  Broken up it would be easier to manage and understand.  It could also be DRYer.